Changelog.md

```diff
@@ -16,6 +16,8 @@
 - Upgrade Docker environment to use Ruby v3.3 (#7185)
 - Upgrade to Rails v7.2 (#7185)
 - Manually specify chromedriver port number in Github actions (#7209)
+- Move Exception message in student model to a localization file
```

Comment:
@anubhajoshi01 oh I didn't notice this last time, but you should also make sure to add the pull request number here (7218) for both entries. This is the standard practice for this changelog.

Reply:
I made the change